src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/msquic_generated.cs

```diff
@@ -929,6 +929,7 @@ internal enum QUIC_PERFORMANCE_COUNTERS
         PATH_FAILURE,
         SEND_STATELESS_RESET,
         SEND_STATELESS_RETRY,
+        CONN_LOAD_REJECT,
```

Comment:
This is related to https://github.com/microsoft/msquic/pull/4120, we should wait until that one merges.

We don't need to wait with MsQuic update before merging, the counter will be 0 until a version which supports it is loaded.